python/notcurses-pydemo

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/python3
```

Comment:
Ahh, I see, the two demo scripts have both been removed. Hrmmm. I'm not crazy about that.

Reply:
I think I moved both of them in to the examples.